.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
+install:
+  - "pip install -r requirements.pip"
+  - "pip install -r requirements-dev.pip"
+  - "pip install coveralls"
+  - "python setup.py install"
+script:
+    - "coverage run --source=vumi_twilio_api `which py.test` vumi_twilio_api"
```

Comment:
Don't know why I can only see Simon dH's comment on the indenting in email and not GitHub.